client/ayon_core/modules/deadline/plugins/publish/submit_houdini_render_deadline.py

```diff
@@ -103,10 +109,14 @@ def get_attribute_defs(cls):
                 minimum=1,
                 maximum=1000
             ),
+            TextDef("group",
+                    default=cls.group,
+                    label="Group Name"
+            ),
```

Comment:
Keep formatting the same please.
```suggestion
            TextDef(
                "group",
                 default=cls.group,
                 label="Group Name"
            ),
```